app/os/main.cpp

```diff
@@ -408,15 +408,17 @@ namespace robot {
 
 	namespace internal {
 
-		auto sleep_timeout = CoreTimeout {};
+		auto inner_state_timeout	  = CoreTimeout {};
+		auto state_transition_timeout = CoreTimeout {};
```

Comment:
small naming suggestion:

```suggestion
		auto timeout_state_internal = CoreTimeout {};
		auto timeout_state_transition  = CoreTimeout {};
```